src/plugins/intel_cpu/src/config.cpp

```diff
@@ -217,29 +212,6 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
             if (!device_id.empty()) {
                 OPENVINO_THROW("CPU plugin supports only '' as device id");
             }
-            OPENVINO_SUPPRESS_DEPRECATED_START
-        } else if (key == InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16) {
```

Comment:
can we also removed https://github.com/openvinotoolkit/openvino/tree/master/src/inference/include/ie/cpu in separate PR?

Reply:
ok, will do it!